enterprise/frontend/src/embedding-sdk/lib/theme/default-component-theme.ts

```diff
@@ -38,6 +38,20 @@ export const DEFAULT_METABASE_COMPONENT_THEME: MetabaseComponentTheme = {
         hoverBackgroundColor: "var(--mb-color-brand)",
       },
     },
+    emptyContent: {
```

Comment:
I think we should also add spacing here, but I don't know if we should basically add padding and margin options for all of these?

Reply:
I think we can keep it minimum until we have more customer needs or requests, that way we're not adding too many theme options.